src/main/java/com/twilio/sdk/resource/instance/ipmessaging/Message.java

```diff
@@ -107,6 +109,15 @@ public String getBody() {
 	}
 
 	/**
+	 * Returns the index of this message in the Channel
+	 *
+	 * @return the index
+	 */
+	public Integer getIndex() {
```

Comment:
i'd be in favor of using primitive return types if possible as it removes any concerns of `null`.